src/pages/search/project_search/ProjectStatusFilter.tsx

```diff
@@ -0,0 +1,78 @@
+import { Box, MenuItem, TextField, Typography } from '@mui/material';
+import { ProjectSearchParameter } from '../../../api/cristinApi';
+import { dataTestId } from '../../../utils/dataTestIds';
+import { useTranslation } from 'react-i18next';
+import { useHistory } from 'react-router';
+import { TFuncKey } from 'i18next';
+import React from 'react';
+
+interface FilterItem {
+  field: string;
+  i18nKey: TFuncKey;
+}
+
+enum Status {
+  ALL = 'ALL',
+  ACTIVE = 'ACTIVE',
+  CONCLUDED = 'CONCLUDED',
+  NOTSTARTED = 'NOTSTARTED',
+}
```

Comment:
Vi har allerede dette i typen til prosjekt, så jeg tenker vi kan slå sammen litt av det.

I `project.types.ts` kan vi legge inn `export type ProjectStatus = 'ACTIVE' | 'CONCLUDED' | 'NOTSTARTED'`, som vi igjen bruker på `status` property til `CristinProject` (i samme fil). På den måten vil filtrene her henge direkte sammen med prosjekttypen vi bruker ellers, og vi trenger ikke å være redd for at dette avviker i fremtiden.

```suggestion
interface FilterItem {
  statusValue: ProjectStatus | '';
  i18nKey: TFuncKey;
}
```

Reply:
Ble mye greier, og litt hit-og-dit her, så vi kan ta en runde på det i morgen om ikke alt gir mening!

Reply:
Elegant!